### ISSUE TITLE
MIPS Instruction: lwx     $t9, $t9($s3) - Unhandled CPU exception (UC_ERR_EXCEPTION),  

### ISSUE THREAD

**Issue by HaToan** [unicorn-engine/unicorn#1314]
```from __future__ import print_function
from unicorn import *
from unicorn.mips_const import *
from capstone import *


MIPS_CODE_EL = b"\x0a\xc8\x79\x7e" # lwx $t9, $t9($s3) 

ADDRESS      = 0x10000


def hook_block(uc, address, size, user_data):
    md = Cs(CS_ARCH_MIPS, CS_MODE_MIPS32 + CS_MODE_LITTLE_ENDIAN)
    print(">>> Tracing instruction at 0x%x, instruction size = 0x%x" %(address, size))
    print(">>> Instruction code at [0x%x] =" %(address))
    tmp = uc.mem_read(address, size)
    print(''.join('{:02x}'.format(x) for x in tmp))
    for i in md.disasm(tmp, address):
        print("0x%x:\t%s\t%s" %(i.address, i.mnemonic, i.op_str))

def hook_code(uc, address, size, user_data):
    print(">>> Tracing instruction at 0x%x, instruction size = 0x%x" %(address, size))

def test_mips_el():
    print("Emulate MIPS code (little-endian)")
    try:
        
        mu = Uc(UC_ARCH_MIPS, UC_MODE_MIPS32 + UC_MODE_LITTLE_ENDIAN)

        mu.mem_map(ADDRESS, 2 * 1024 * 1024)

        mu.mem_write(ADDRESS, MIPS_CODE_EL)

        mu.reg_write(UC_MIPS_REG_1, 0)

        mu.hook_add(UC_HOOK_BLOCK, hook_block)

        mu.hook_add(UC_HOOK_CODE, hook_code)

        mu.reg_write(UC_MIPS_REG_S3, 0x10000)
        
        mu.emu_start(ADDRESS, ADDRESS + len(MIPS_CODE_EL))

        print(">>> Emulation done. Below is the CPU context")

        t9 = mu.reg_read(UC_MIPS_REG_T9)
        print(">>> R1 = 0x%x" %t9)

    except UcError as e:
        print("ERROR: %s" % e)


if __name__ == '__main__':
    print("=" * 27)
    test_mips_el()
```

```
Emulate MIPS code (little-endian)
>>> Tracing instruction at 0x10000, instruction size = 0x4
>>> Instruction code at [0x10000] =
0ac8797e
0x10000:        lwx     $t9, $t9($s3)
>>> Tracing instruction at 0x10000, instruction size = 0x4
**ERROR: Unhandled CPU exception (UC_ERR_EXCEPTION)**
```

The error CPU exception  when I emulates `lwx` instructions .

**Comment by wtdcode** [comment]
Link to #1449

**Comment by wtdcode** [comment]
Fixed in 2d043d387dc44ea71eaa6b5b6efeeff2c9e5d9b4

You have to enable DSP to emulate `lwx` instruction.
